docs/csv-export.md

```diff
@@ -0,0 +1,133 @@
+---
+sidebar_position: 9
+---
+
+# CSV Export
+
+Opencost supports CSV export to a local file, Azure Blob Storage, AWS S3, and Google Cloud Storage.
+
+The data is aggregated at a day level and exported every day after 00:10 UTC.
+The first export includes all available data, while subsequent launches only export data for dates not present in the existing export file.
+
+Check example of [export file](/export-sample.csv).
+
+
+## Configuration
+
+
+To enable export, set the `COST_EXPORT_FILE` environment variable to the path of the file. The file can be a local file or a storage object in one of the clouds. Here are some usage examples:
+
+| Provider             | Value                                                                        |
+|----------------------|------------------------------------------------------------------------------|
+| Azure Blob Storage   | `https://azblobaccount.blob.core.windows.net/containername/path/to/file.csv` |
+| AWS S3               | `s3://bucketname/path/to/file.csv `                                          |
+| Google Cloud Storage | `gs://bucket-name/path/to/file.csv`                                          |
+| Local File           | `/path/to/file.csv `                                                         |
+
+
+
+### Export data to a persistent volume.
+
+There is an example of a change required for the opencost deployment to export data to a persistent volume.
+
+```yaml
+---
+# Create a persistent volume claim for storing the data
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: opencost-export
+  namespace: opencost
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi # amount of required storage is based on cluster size and type of the workload running on it, adjust if required
+---
+# Update the opencost deployment
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: opencost
+  # ...
+spec:
+  # ...
+  template:
+    # ...
+    spec:
+      # ...
+      securityContext:
+        fsGroup: 1001 # opencost is running as a non-root user, this gives container permission to write to the pvc
```

Comment:
Without this change opencost container don't have permissions to write into volume mount.

I'm not sure if it's the best approach though.